action.rb

```diff
@@ -1,25 +1,35 @@
 class Action
   attr_reader :headers, :body, :request
 
-  def initialize &block
+  def initialize(&block)
     @block = block
-    @status = 200
-    @headers = {'Content-Type' => 'text/html'}
-    @body = ''
+    @headers = {'Content-Type' => 'application/json'}
   end
 
-  def status value = nil
-    value ? @status = value : @status
+  def status(value = nil)
+    value ? @status = value : @status || 200
```

Comment:
`value ? (@status = value) : (@status || 200)` it's probably clearer to read as an if/else given there's an assignment in one branch and another branch in the other. i like ternaries but they're best reserved for single-term expressions instead of complex ones. this is something that frequently bewilders newbies when they aren't fully literate yet.